tests/core_tests/state_test.py

```diff
@@ -0,0 +1,15 @@
+import unittest, logging
+import src.core.state
+
+DEBUG = False
+
+
+class StateTestCases(unittest.TestCase):
+    def test_state(self):
+        pass
+
+    # TODO
+
```

Comment:
Let's actually implement this!

Reply:
Just need to test whether a state object initializes, can init from a dict, and that the to_array method works as expected